src/Umbraco.Web/PublishedCache/XmlPublishedCache/XmlPublishedContent.cs

```diff
@@ -432,7 +433,20 @@ private void InitializeNode()
 		    }
 
             //lookup the content type and create the properties collection
-            contentType = getPublishedContentType(PublishedItemType.Content, docTypeAlias);
+            try
+            {
+                contentType = getPublishedContentType(PublishedItemType.Content, docTypeAlias);
+
+            }
+            catch (Exception e)

```

Comment:
This needs to catch the correct exception that you are expecting, not just the generic exception